src/main/webapp/WEB-INF/views/admin/oss/ossBulkReg-js.jsp

```diff
@@ -114,8 +163,35 @@
             autowidth: true,
             gridview: true,
             height: 'auto',
-            pager: '#pager'
-        });
+            pager: '#pager',
+            autoencode: true,
+            editurl:'clientArray',
+            recordpos:'right',
+            toppager:true,
+            loadonce:false,
+            cellsubmit : 'clientArray',
+            ignoreCase: true,
+            multiselect: true,
+
+            loadComplete: function(data) {
+                _mainLastsel = -1;
+            },
+            ondblClickRow: function(rowid,iRow,iCol,e) {
+                if(rowid) {
+                    if (rowid != _mainLastsel) {
+                        $("#list").jqGrid('saveRow', _mainLastsel);
+                    }
+
+                    editable()
+
+                    $("#list").jqGrid('editRow', rowid);
+                    _mainLastsel = rowid;
+
+                    var nextCol = $("#list").jqGrid('getGridParam', 'colModel')[iCol].name
+                    var nextRow = rowid
+                    $('#' + nextRow + "_" + nextCol).focus();
+            }
+        }});
```

Comment:
How about change indentation of code likes below?
```javascript
    ondblClickRow: function(rowid,iRow,iCol,e) {
        if(rowid) {
            if (rowid != _mainLastsel) {
                $("#list").jqGrid('saveRow', _mainLastsel);
            }
            editable()
            $("#list").jqGrid('editRow', rowid);
            _mainLastsel = rowid;
            var nextCol = $("#list").jqGrid('getGridParam', 'colModel')[iCol].name
            var nextRow = rowid
            $('#' + nextRow + "_" + nextCol).focus();
        }
    }
});
```

Reply:
thank you for your review! I fixed commit right before